### PR TITLE
Changelog for v2.6.0

### PR DESCRIPTION
**Pull request by mauriciopoppe**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
/kind cleanup

**What this PR does / why we need it**:
Changelog for v2.6.0

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/cc @xing-yang @jingxu97

### CODE REVIEW

File: CHANGELOG/CHANGELOG-2.6.0.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v2.5.0
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Corrects the handler used to serve http requests, the /healthz and /debug/ endpoints are available now. (#190, @cnmcavoy)
```
Comment (xing-yang) [inline CHANGELOG/CHANGELOG-2.6.0.md:11]: Can you format it like this?
https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/master/CHANGELOG/CHANGELOG-6.1.md

Comment (mauriciopoppe) [reply CHANGELOG/CHANGELOG-2.6.0.md]: done, please take a look again

**Pull request merged**